src/c++/perf_analyzer/test_request_rate_manager.cc

```diff
@@ -862,8 +861,57 @@ TEST_CASE("Request rate - Shared memory methods")
   {
     params.shared_memory_type = CUDA_SHARED_MEMORY;
     TestRequestRateManager trrm(
-        params, mip.mock_model_parser_, is_sequence, is_decoupled,
-        use_mock_infer);
+        params, is_sequence, is_decoupled, use_mock_infer);
+    std::shared_ptr<MockInferDataManager> mock_infer_data_manager{
+        std::make_shared<MockInferDataManager>(
+            params.batch_size, params.shared_memory_type,
+            params.output_shm_size, mip.mock_model_parser_, trrm.factory_,
+            mip.mock_data_loader_)};
+    trrm.infer_data_manager_ = mock_infer_data_manager;
+    trrm.parser_ = mip.mock_model_parser_;
+    trrm.data_loader_ = mip.mock_data_loader_;
+    trrm.InitManager(
+        params.string_length, params.string_data, params.zero_input,
+        params.user_data);
+
+    expected_stats.num_unregister_all_shared_memory_calls = 1;
+    expected_stats.num_register_cuda_shared_memory_calls = 1;
+    trrm.CheckSharedMemory(expected_stats);
+  }
+
+  SUBCASE("No shared memory usage")
```

Comment:
It looks like this subcase (and cuda shared memory usage below) is duplicated and the result of a bad merge?

Reply:
Removed